packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -20,11 +20,14 @@ import { useRouter } from '../../router';
 import type { PropsOfComponent, ThemableCssProp } from '../../styledSystem';
 import { OrganizationActionList } from './OtherOrganizationActions';
 
-type OrganizationSwitcherPopoverProps = { close: () => void } & PropsOfComponent<typeof PopoverCard.Root>;
+type OrganizationSwitcherPopoverProps = {
+  close?: (open: boolean | ((prevState: boolean) => boolean)) => void;
+} & PropsOfComponent<typeof PopoverCard.Root>;
 
 export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, OrganizationSwitcherPopoverProps>(
   (props, ref) => {
-    const { close, ...rest } = props;
+    const { close: undefinedClose, ...rest } = props;
```

Comment:
I'm assuming you named it `undefinedClose` because it can be potentially undefined? Maybe use `unsafeClose` instead?